packages/SwingSet/src/kernel/kernel.js

```diff
@@ -966,7 +980,7 @@ export default function buildKernel(
       const vatKeeper = kernelKeeper.provideVatKeeper(vatID);
 
       // deliveries cause garbage, garbage needs collection
-      const { decrementReapCount } = status;
+      const { decrementReapCount, terminate } = status;
```

Comment:
nit: I'm -0 on merging the later destructuring up here, I guess it's a minor aesthetic thing. There are three sections here (`scheduleReap`, meter deduction, check termination), each one starts with unpacking the necessary bits of `status` and then interpreting it somehow. Unpacking `terminate` here but then not using it until the third clause makes it look (to me) that `terminate` is somehow needed for this first section, but it isn't.

(an admission: the only reason I used any destructuring at all is because that made the lines short enough to keep `prettier` from newlining the code into a bunch of uselessly short lines, making the otherall function no longer fit on a single page. If it wasn't going to do that, I'd have used `if (status.decrementReapCount && ...` instead)

Maybe move all the destructuring, from all three sections, up to sit right after `const vatKeeper = ..`? Except that having more than two or three things destructured at once will *also* cause `prettier` to splat expand the statement into a gratuitious number of lines. Can't win.

Reply:
Actually, this happened as the result of another change I made that referenced `terminate` earlier than it had been, and then that change ended up getting backed out in the course of finishing up the work on this item.  So we can put it back the way it was if that's your preference.